src/poetry/console/commands/group_command.py

```diff
@@ -107,3 +111,36 @@ def project_with_activated_groups_only(self) -> ProjectPackage:
         return self.poetry.package.with_dependency_groups(
             list(self.activated_groups), only=True
         )
+
+    def validate_group_options(self, group_options: dict[str, set[str]]) -> bool:
+        """
+        Currently raises en error if it detects that a group is
+        not part of pyproject.toml
+
+        Can be overridden to adapt behavior.
```

Comment:
```suggestion
```

I'd leave this away. In contrast to `default_groups`, I don't think that this one will often be overridden.

Reply:
Changed in Changed in d9aaafe64f8602e6832a2f098275dc9ed0ff7c56